src/components/utils/caseUtils.ts

```diff
@@ -61,26 +61,31 @@ export function useDownloadUtils() {
         formatParameters: {
             [parameterName: string]: any;
         },
-        abortController: AbortController
+        abortController: AbortController,
+        fileName: string
     ): Promise<void> => {
         try {
+            // if no fileName is provided, the case name will be used
+            const autoName: boolean = fileName === undefined || fileName === '';
             const result = await fetchConvertedCase(
                 caseElement.elementUuid,
-                caseElement.elementName,
+                autoName ? caseElement.elementName : fileName,
```

Comment:
for default value in JS we can use || as following
fileName || caseElement.elementName

Reply:
Great !